.github/workflows/build.yml

```diff
@@ -31,6 +31,7 @@ jobs:
           cd userday2022/functional_sensitivity && pdflatex -halt-on-error functional_sensitivity.tex && pdflatex functional_sensitivity.tex && cd -
           cd userday2022/userday2022relhi/ && pdflatex -halt-on-error userday2022relhi.tex && pdflatex userday2022relhi.tex && cd -
           cd userday2022/ud2022persalys && pdflatex -halt-on-error UsersDay-2022-PERSALYS.tex && pdflatex UsersDay-2022-PERSALYS.tex && cd -
+          cd developer/python_function && bash build.sh && cd -
```

Comment:
this should be moved in the for loop at line 18 for the developer dir
for this the main .tex file must have the same name as the subdir (python_function)
then your build.sh can be dropped

Reply:
Ok, fixed.